website/templates/admin/admin_sensors.html.mako

```diff
@@ -8,6 +8,8 @@
     <h1 class="title">Admin sensors</h1>
 </%block>
 
+<%include file="add_sensor.html.mako" />
```

Comment:
This one is not going up one level?

Reply:
For some odd reason it did not work when added "../" in front... But that shouldn't be a problem if you do a mako function for it.

Reply:
If you look at the "%inherit" at the top of the admin page, it does not have a "../" either, but in `accessor/sensor.html/mako` it does.